features/events/api/pom.xml

```diff
@@ -33,8 +33,8 @@
   </build>
   <dependencies>
     <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
You'll also need to update the feature definition to depend on commons-lang3:
    https://github.com/OpenNMS/opennms/blob/foundation-2017/container/features/src/main/resources/features.xml#L612

(This is likely why some of the current smoke tests are failing)